### DISCUSSION TITLE
Alt+Tab in Windows 7 breaks Standard Notes

### DISCUSSION THREAD

**Discussion by babbysfirstgithub** [standardnotes/forum#3120] [General]
When a note has focus (i.e. after typing) and you Alt-Tab out of SN, once you come back to SN, it's not possible to type anything. Moving the cursor (arrow keys or mouse clicks) is also not possible, as is changing notes.
The problem resolves after waiting for ~ 10-15 seconds once SN has focus again.

Happens on Windows 7, all versions of SN, for at least 1 year now.

This seriously was the one factor keeping me from paying for pro.

**Comment by babbysfirstgithub** [comment]
Addendum: Two details I found
1) Sometimes a popup will show after a few seconds of unsuccessfully trying to regain input focus: "Attempting to save an item while the application is hidden. To protect data integrity, please refresh the application window and try again."
2) After running SN for 30+ minutes & a few focus changes, even if it keeps working, there is a significant input lag of ~ 200-500ms from keystroke to the letter showing up.

**Comment by effieeee** [comment]
Hi @babbysfirstgithub, may I first ask what version of the desktop app you have installed on your device at the moment? Do you also experience this issue while using the web app? (app.standardnotes.com)

**Comment by babbysfirstgithub** [comment]
Hi! Version 3.100.17, but this problem has existed for at least 1 year now. As for the web app, no issue there. 

I just realized a little cue that might help in pinning down the bug. When alt-tabbing back to SN and the problem occurs, the red autocorrect underlining remains visible under most of my note text for 10-15 seconds (as long as the focus problem persists), then it disappears just a split second before focus is regained.

**Comment by dec0dOS** [comment]
The same annoying thing appears with CMD+H on macOS 13.1.

`Attempting to save an item while the application is hidden. To protect data integrity, please refresh the application window and try again.`

That's kills multitasking.
Is there any way do disable it?

**Comment by babbysfirstgithub** [comment]
Sometimes I get the same message; but when it appears, it usually happens either without any freezing or right after a 10-20s freeze, right when the window becomes "unlocked" again.
